packages/extension-chakra-storefront/src/components/links-list/index.jsx

```diff
@@ -6,7 +6,13 @@
  */
 import React from 'react'
 import PropTypes from 'prop-types'
-import {Box, List, ListItem, Heading, useMultiStyleConfig} from '@chakra-ui/react'
+import {
+    Box,
+    Heading,
+    List,
+    // hooks,
```

Comment:
super nit: I was reading this as we commented out "hooks" from chakra-ui 😄 
```suggestion
    // hooks
```